src/main/java/entities/Timer.java

```diff
@@ -42,7 +43,7 @@ public class Timer {
 // constructor
     /**
      * Constructs a Timer given the starting duration
-     * @param duration The duration for the Timer (how long of a Timer session do we want)
+     * @param duration The duration for the Timer, how long of a Timer session do we want
      */
     public Timer(Duration duration) {
 
```

Comment:
Is something wrong here? Looks like startDuration and remainDuration get assigned the same duration

Reply:
(line 50,51)

Reply:
During the time of creation of a new timer object, both those values are the same. Later, as time starts to run out, I subtract it from remainig duration.